docs/product/explore/session-replay/index.mdx

```diff
@@ -1,41 +1,5 @@
 ---
 title: "Session Replay"
 sidebar_order: 70
-description: "Use Session Replay to get video-like reproductions of user interactions to improve your website or app experience."
+description: "Use Session Replay to get reproductions of user sessions to improve your app experience."
```

Comment:
```suggestion
description: "Learn about Sentry's Session Replay feature which allows you to view video-like reproductions of user sessions so you can better understand user impact."
```

Reply:
@lizokm This is the parent page for both web & mobile, so we can't actually say "video-like reproductions". The mobile replays are videos 🫢